boltz/ws.go

```diff
@@ -0,0 +1,161 @@
+package boltz
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/url"
+	"time"
+
+	"github.com/BoltzExchange/boltz-client/logger"
+	"github.com/gorilla/websocket"
+)
+
+const reconnectInterval = 15
```

Comment:
```suggestion
const reconnectInterval = 15 * time.Second
```

Reply:
I would keep it like that so I can use it in the log

Reply:
It knows how to print it, nevermind